dali/pipeline/executor/executor_test.cc

```diff
@@ -576,30 +506,24 @@ TYPED_TEST(ExecutorSyncTest, TestPrefetchedExecution) {
   exe->RunMixed();
   exe->RunGPU();
 
-  auto status_1 = barrier_future_1.wait_for(std::chrono::seconds(5));
-  ASSERT_EQ(status_1, std::future_status::ready);
-  ASSERT_EQ(cb_counter, 1);
   src_op->SetDataSource(tl2);
   exe->RunCPU();
   exe->RunMixed();
   exe->RunGPU();
 
   DeviceWorkspace ws;
-  exe->Outputs(&ws);
+  exe->ShareOutputs(&ws);
   ASSERT_EQ(ws.NumOutput(), 1);
   ASSERT_EQ(ws.NumInput(), 0);
   ASSERT_TRUE(ws.OutputIsType<GPUBackend>(0));
   test::CheckResults(ws, batch_size, 0, tl);
+  exe->ReleaseOutputs();
```

Comment:
Why this change?

Reply:
Wanted to explicitly at this point ReleaseOutputs() before next run.
From functionality perspective doesn't change the semantics, since next exe->Outputs() would call that for but after next run.

Reply:
NVM can probably revert that change. It is pointless here.

Reply:
Actually, a call to Otuputs calls Release first, then Share. But whatever - if it works...